permissions/plugin-dotnet-sdk.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "dotnet-sdk"
+github: "jenkinsci/dotnet-sdk-plugin"
+paths:
+- "io/jenkins-ci/plugins/dotnet-sdk"
```

Comment:
```suggestion
- "io/jenkins/plugins/dotnet-sdk"
```

See https://github.com/jenkinsci/dotnet-sdk-plugin/blob/master/pom.xml#L7

Reply:
My bad - must have been looking at the parent plugin or something. And there I was, thinking I’d triple-checked things...